test_runner/fixtures/compare_fixtures.py

```diff
@@ -159,6 +159,45 @@ def record_duration(self, out_name):
         return self.zenbenchmark.record_duration(out_name)
 
 
+class RemoteCompare(PgCompare):
+    """PgCompare interface for a remote postgres instance."""
+    def __init__(self, zenbenchmark, remote_pg: RemotePostgres):
+        self._pg = remote_pg
+        self._zenbenchmark = zenbenchmark
+
+        # Long-lived cursor, useful for flushing
+        self.conn = self.pg.connect()
+        self.cur = self.conn.cursor()
+
+    @property
+    def pg(self):
+        return self._pg
+
+    @property
+    def zenbenchmark(self):
+        return self._zenbenchmark
+
+    @property
+    def pg_bin(self):
+        return self._pg.pg_bin
+
+    def flush(self):
+        self.cur.execute("checkpoint")
```

Comment:
I think it's better to either `pass` or `raise NotImplementedError`. Tests that call this function expect it to checkpoint the pageserver, and AFAIK that won't happen by calling checkpoint on compute.

Luckily pgbench doesn't need flush. For the tests that do need this, we can initially run them against remote by just running them for longer and relying on pageserver to checkpoint itself. Maybe at some point we can add a pageserver connection to the env so they can checkpoint it, but that goes against the purpose of testing in a prod-like environment. Not sure yet.

Reply:
Good point, you would expect all these functions to connect to the remote pageserver and flush or get size from there. I changed this to `pass`, and added TODO comments to note that it hasn't been implemented.